WalletWasabi.Fluent/ViewModels/MainViewModel.cs

```diff
@@ -55,7 +56,7 @@ public MainViewModel()
 		_isDialogScreenEnabled = true;
 		_isFullScreenEnabled = true;
 
-		_statusIcon = new StatusIconViewModel();
+		_statusIcon = new StatusIconViewModel(new StatusChecker(new TorNetwork(new UriBasedStringStore(), new IssueParser()), TimeSpan.FromSeconds(20), new NewThreadScheduler()));
```

Comment:
Yes! Since status won't change frequently, I guess this can be increased up to 1 hour or more. What do you think?

Reply:
6 hours looks OK to me. The initial check will always be there:
- If user is having a long sessions (CJs and so) it will be enough. Updates happen rarely.
- Quick sessions are covered. The update is checked always start.